content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -139,6 +139,7 @@ ServiceNow Variable Name | Terraform Enterprise Variable
 --|--
 `tf_var_VARIABLE_NAME` | Terraform Variable: `VARIABLE_NAME`
 `tf_env_ENV_NAME` | Environment Variable: `ENV_NAME`
+`sensitive_tf_var_VARIABLE_NAME` | Sensitive Terraform Variable (Write Only): `VARIABLE_NAME`
```

Comment:
Are you considering adding an equivalent setting for sensitive environment variables? I think those might be more widely used, since most providers use env vars for their credentials.